app/serializers/user_serializer.rb

```diff
@@ -46,10 +48,13 @@ class UserSerializer
   end
 
   attributes :induction_programme_choice do |user|
+    find_school_cohort(user)&.induction_programme_choice
+  end
+
+  def self.find_school_cohort(user)
     @school_cohorts ||= SchoolCohort.all.to_a
```

Comment:
Leave for now, but I'd be interested to know if we actually get a performance boost from reading potentially ~20k records (and growing every year) into memory to save 100 queries

Reply:
Hmm, not sure about performance, but I am pretty sure it saves us loads of very similar logs